index.html

```diff
@@ -41,7 +50,196 @@ <h1 class="hero_head">Hey there. I'm Jose Delphin.<br> <span>I'm a software deve
       <a id="down_icon" href="#"><img src="assets/images/down.png" alt="Icon for moving"></a>
     </section>
   </div>
- 
+  <header>
+    <h2 id="recent-works" class="works_head">My Recent Works </h2>
+    <img class="center" src="assets/images/Indicator.png" alt="Design line">
+
+  </header>
+
+  <!-- main grid container -->
+  <section id="project_section" class="grid_container">
+    <ul class="flex_container_secondary">
+      <li class="nested_flexbox">
+        <div class="items_space">
+
+        </div>
+        <h3>Multi-Post Stories</h3>
+        <h3>Gain + Glory</h3>
+        <ul>
+          <li>Ruby on rails</li>
+          <li>css</li>
+          <li>JavScript</li>
+          <li>html</li>
+        </ul>
+        <div class="errorButton">
+          <button class="form_button" type="submit">See project</button>
```

Comment:
Good job making this element a `button` tag, but also the different states (hover, active and disable) of the button should be styled as well.

<img width="440" alt="states" src="https://user-images.githubusercontent.com/84930355/205422931-582638e9-cdc3-4ad9-91ab-88811b6dfae6.png">


Consider including this in this project to make the action on this website more interactive. Also, I suggest you change the button type from `submit` to `button` because you don't want to trigger a submit event as we are not in a `form`

Reply:
thank you for suggestion